src/platform/utilities/feature-toggles/index.jsx

```diff
@@ -2,7 +2,7 @@
 import { FlipperClient } from 'platform/utilities/feature-toggles/flipper-client';
 import environments from 'platform/utilities/environment';
 
-const { fetchToggleValues } = new FlipperClient();
+const { fetchToggleValues } = new FlipperClient({ host: environments.API_URL });
```

Comment:
no longer uses the default value- uses the `environments` helper to set the hostname